phonebook-ui/angular.json

```diff
@@ -123,5 +123,8 @@
       }
     }
   },
-  "defaultProject": "phonebook-ui"
-}
+  "defaultProject": "phonebook-ui",
+  "cli": {
+    "analytics": false
```

Comment:
what is it?

Reply:
Stop angular cli asking for collecting analytics when I use ng serve first time